### PR TITLE
Update usher to not skip Mac OSX build

### PR DESCRIPTION
**Pull request by AngieHinrichs**
I asked for help with a testing issue on the gitter channel and @dpryan79 suggested making a PR, so here it is, thanks!

I made a branch in which I deleted a line from recipes/usher/meta.yaml that skips the build on Mac.  I tried to test locally on my MacBook (Catalina 10.5.7) by running these commands:
```
cd bioconda-recipes
./bootstrap.py --no-docker /tmp/miniconda
source ~/.config/bioconda/activate
bioconda-utils build --git-range master
```
It runs successfully through fetching the source tar for usher, but then hangs indefinitely when fetching the source file for usher's dependency TBB:
```
12:17:07 BIOCONDA INFO (ERR) INFO conda_build.source:download_to_cache(83): Downloading https://github.com/yatisht/usher/archive/v0.3.1.tar.gz
12:17:11 BIOCONDA INFO (OUT) Success
12:17:11 BIOCONDA INFO (ERR) INFO:conda_build.source:Success
12:17:11 BIOCONDA INFO (ERR) INFO conda_build.source:download_to_cache(94): Success
12:17:11 BIOCONDA INFO (OUT) Extracting download
12:17:11 BIOCONDA INFO (ERR) INFO:conda_build.source:Downloading source to cache: 2019_U9_3f5ea81b9c.tar.gz
12:17:11 BIOCONDA INFO (OUT) Downloading source to cache: 2019_U9_3f5ea81b9c.tar.gz
12:17:11 BIOCONDA INFO (OUT) Downloading https://github.com/oneapi-src/oneTBB/archive/2019_U9.tar.gz
12:17:11 BIOCONDA INFO (ERR) INFO conda_build.source:download_to_cache(69): Downloading source to cache: 2019_U9_3f5ea81b9c.tar.gz
12:17:11 BIOCONDA INFO (ERR) INFO:conda_build.source:Downloading https://github.com/oneapi-src/oneTBB/archive/2019_U9.tar.gz
12:17:11 BIOCONDA INFO (ERR) INFO conda_build.source:download_to_cache(83): Downloading https://github.com/oneapi-src/oneTBB/archive/2019_U9.tar.gz
..................................................................................
```

It is reproducible; I started the process last night, and this morning it was still outputting dots.  I stopped it, did `rm -rf /tmp/miniconda`, repeated the above commands, and the same thing happened.

@BiocondaBot please update

### CODE REVIEW

**Comment by bgruening** [comment]
@AngieHinrichs see here: https://github.com/bioconda/bioconda-recipes/pull/29062

**Comment by AngieHinrichs** [comment]
Thanks again @dpryan79 for the help on gitter!  

@BiocondaBot please add label
@BiocondaBot please fetch artifacts

**Comment by AngieHinrichs** [comment]
Seems this didn't take, trying again:

@BiocondaBot please add label

**Comment by dpryan79** [comment]
@AngieHinrichs The bot only looks at the very beginning of comments, so if you type something before "@BiocondaBot please blah" it'll ignore it.

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

I still wonder why conda-build fails at downloading the tarball.

File: recipes/usher/meta.yaml [hunk 1]
```diff
@@ -30,8 +26,8 @@ requirements:
     - python
     - mafft
     - biopython
+    - boost-cpp
     - ucsc-fatovcf
-    - {{ pin_compatible('boost-cpp', max_pin='x.x.x') }}
```
Comment (bgruening) [inline recipes/usher/meta.yaml:34]: @dpryan79 why removing that and including the unpinned?

**Pull request merged**